blog/2023-03-16-financial-revolution-powered-by-the-blockchain/financial-revolution-powered-by-the-blockchain.md

```diff
@@ -0,0 +1,103 @@
+
+**4. Lending:**
+
+You need a credit score to access credit, but you need credit to have a credit score. Blockchain has a solid potential to solve that problem. For example, Moola Market, a lending protocol on Celo, recently partnered with Kotani Pay, an off-ramp service built on top of Celo, to do a pilot where farmers in Nanyuki, Kenya, could have access to a short-term loan. The idea was to award loans to members with a strong standing in the community, not necessarily a credit score, paving the way for smallholder farmers to have the chance to have access to cheap and efficient financial services. What if we could scale such initiatives?
+
+**5. Exchanging Value:**
+
+Blockchain technology cuts settlement times from all transactions from days to a matter of seconds. This allows the excluded population to participate in trading. The unbanked can effectively trade on ubeswap, a decentralized exchange built on Celo to buy and sell different assets, creating financial freedom and inclusion.
+
+**6. Funding:**
+
+What would it look like to democratize access to investing in enterprises and venture funding? The VC world was traditionally only meant for accredited investors, giving the top 1% the power to invest in the ideas they see fit to change the world. Additionally, you needed intermediaries to get matched with investors. The blockchain automates these processes, thus enabling new models of peer-to-peer financing and making the process more transparent and efficient.
+
+**7. Accounting for Value**
+
+The accounting industry is a multi-billion dollar industry that the big four accounting firms have majorly controlled. Since the blockchain is a distributed, append-only digital ledger, new accounting methods can leverage blockchain security, making financial reporting transparent and occurring in real-time; schemes like Enron (Enron's leadership fooled regulators with fake holdings and off-the-books accounting practices)
+
+Would be impossible to pull off.
+
+**8. Risk Management and Insurance:**
+
+Uncertainty, as described by Nicholas Nassim Taleb, is bound to happen. Thus, human activity should be directed towards protecting oneself from uncertain loss. Blockchain technology supports decentralized insurance, making the insurance industry far more transparent. The insurer can leverage the blockchain to determine  a person’s reputational score based on reputational attributes. These can provide insurers more clarity allowing them to make more informed decisions.
+
+**Conclusion​**
+
+Blockchain technology may go through hiccups, and we may even fail, but I firmly believe that with the insights we have gathered about blockchain technology; we will iterate and reinvent it to serve its true purpose creating sustainable, inclusive communities that have a solid voice. With each leap forward, we will slowly but surely usher in the second era of democracy.
+
+**Next Steps​**
```

Comment:
Use subheadings (H2: ##) to break down your explanations within the body of the tutorial.